spec/system/task_spec.rb

```diff
@@ -1,24 +1,22 @@
 require 'rails_helper'
-
 RSpec.describe 'Task', type: :system do
+  let(:project) { FactoryBot.create(:project) }
```

Comment:
`FactoryBot.`という記載は省略できるので調べてみてください！
```suggestion
  let(:project) { create(:project) }
```